app/service/data_svc.py

```diff
@@ -201,3 +198,22 @@ def __init__(self, dao):
 
     async def update(self, table, key, value, data):
         await self.dao.update(table, key, value, data)
+
+
```

Comment:
remove the extra blank line (should be 1 blank line between functions) add 1 blank line under your comment and change the comment from """ SUPPORT """ to """ PRIVATE """ to match what we have elsewhere - and move it to the bottom of this class.

Reply:
Got it.